step-certificates/examples/certificate_authority_single_instance/values.yml.tpl

```diff
@@ -0,0 +1,130 @@
+kind: StatefulSet
+replicaCount: 1
+autocert:
+  enabled: false
+bootstrap:
+  enabled: false
+ca:
+  db:
+      enabled: true
+      persistent: false
+inject:
+  enabled: true
+  config:
+    files:
+      ca.json:
+        root: /home/step/certs/root_ca.crt
+        federateRoots: []
+        crt: /home/step/certs/intermediate_ca.crt
+        key: /home/step/secrets/intermediate_ca_key
+        address: 0.0.0.0:9000
+        dnsNames: ${CA_DNS_NAMES}
+        logger:
+          format: json
+        db:
+          type: badger
+          dataSource: /home/step/db
+        authority:
+          claims:
+            minTLSCertDuration: 5m
+            maxTLSCertDuration: 24h
+            defaultTLSCertDuration: 24h
+            disableRenewal: false
+            minHostSSHCertDuration: 5m
+            maxHostSSHCertDuration: 1680h
+            defaultHostSSHCertDuration: 720h
+            minUserSSHCertDuration: 5m
+            maxUserSSHCertDuration: 24h
+            defaultUserSSHCertDuration: 24h
+          provisioners:
+            - type: ACME
+              name: acme
+              forceCN: true
+              claims: {}
+            - type: SSHPOP
+              name: sshpop
+              claims:
+                enableSSHCA: true
+            - type: JWK
+              name: ansible_automation_token
```

Comment:
Can we templatize this, and default it to admin

Reply:
done